Kitodo/src/main/java/org/kitodo/production/interfaces/activemq/MapMessageObjectReader.java

```diff
@@ -110,6 +114,53 @@ public String getMandatoryString(String key) throws JMSException {
         return mandatoryString;
     }
 
+    /**
+     * Fetches a {@code Collection<Integer>} from a MapMessage. This is a loose
+     * implementation for an optional object with optional content. The
+     * collection content is filtered through {@code toString()} and split on
+     * non-digits, dealing generously with list variants and separators. If not
+     * found, returns an empty collection, never {@code null}.
+     *
+     * @param key
+     *            the name of the set to return
+     * @return the set requested
+     * @throws JMSException
+     *             can be thrown by MapMessage.getObject(String)
+     */
+    public Collection<Integer> getCollectionOfInteger(String key) throws JMSException {
+        Collection<String> collectionOfString = getCollectionOfString(key);
+        List<Integer> collectionOfInteger = collectionOfString.stream()
+                .flatMap(string -> Arrays.stream(string.split("\\D+"))).filter(Strings::isNotBlank)
+                .map(Integer::valueOf).collect(Collectors.toList());
+        return collectionOfInteger;
```

Comment:
```suggestion
        return collectionOfString.stream()
                .flatMap(string -> Arrays.stream(string.split("\\D+"))).filter(StringUtils::isNumeric)
                .map(Integer::valueOf).collect(Collectors.toList());
```
No need to assigne the result of the stream operation to a variable and then return the variable in the next line - just inline it, e.g. directly return the result.

Reply:
I used to do it this way, but then I should do it the other way around. Now you want me to do it the other way around again. No problem, I just need to know.